services/movie_session.py

```diff
@@ -6,48 +6,45 @@
 def create_movie_session(movie_show_time: datetime,
                          movie_id: int,
                          cinema_hall_id: int) -> MovieSession:
-    if movie_show_time and movie_id and cinema_hall_id:
-        cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
-        movie = Movie.objects.get(id=movie_id)
-        movie_session = MovieSession(
-            show_time=movie_show_time,
-            cinema_hall=cinema_hall,
-            movie=movie
-        )
-        movie_session.save()
-        return movie_session
+    cinema_hall_id = CinemaHall.objects.get(id=cinema_hall_id)
+    movie_id = Movie.objects.get(id=movie_id)
+    movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall_id,
+        movie=movie_id
+    )
+    return movie_session
```

Comment:
btw, see, it is the same code:

```python
movie_session = MovieSession.objects.create(
        show_time=movie_show_time,
        cinema_hall_id=cinema_hall_id,
        movie_id=movie_id
    )
```

Reply:
so you don't need to overload your db with 
```python
cinema_hall_id = CinemaHall.objects.get(id=cinema_hall_id)
movie_id = Movie.objects.get(id=movie_id)
```

Reply:
as foreignKey saves not a full object but a primary key only. By default primary key is an ID. In `create_movie_session` you take `movie_id` - the id of `Movie` class object. So 
Bad example of ForeignKey saving:
```python
movie_session = MovieSession.objects.create(
        show_time=movie_show_time,
        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
        movie=Movie.objects.get(id=movie_id)
    )
```

Good example:
```python
movie_session = MovieSession.objects.create(
        show_time=movie_show_time,
        cinema_hall_id=cinema_hall_id,
        movie_id=movie_id
    )
```

Reply:
as in the second example, you make 1 request to DB, at the first - 3 requests. Feel the difference now?

Reply:
Thanks for the intelligible explanation! I probably did not quite correctly understand the conditions of the task, I thought that any data could come from outside, and before climbing into the database, they need to be checked whether there is such an id or not.

Reply:
I understand that the way you did it looks much more logical. But in this case, other skills and understanding need to be developed here. + in Django, many things are already done so that errors are handled immediately